### ISSUE TITLE
Enable GCP APIs in `sc add-gcp-broker`

### ISSUE THREAD

**Issue by mihnjong** [GoogleCloudPlatform/k8s-service-catalog#131]
As of now, we enable deployment manager API and service broker API in `sc add-gcp-broker`. We also need to enable other APIs that we support in GCP broker.

The list includes BigQuery, Bigtable, IAM, ML Engine, Pub/Sub, Spanner, SQL, and Storage.

**Comment by mihnjong** [comment]
Maybe, we need to enable all APIs in parallel in case that it might take too long.

**Comment by martinmaly** [comment]
what is the rationale for a flag? Is there harm in (presumably idempotently) enabling the API mutliple times?

**Comment by mihnjong** [comment]
6 APIs need to be added:

`gcloud services enable iam.googleapis.com` for IAM and all other bindings
`gcloud services enable bigtableadmin.googleapis.com` for Bigtable
`gcloud services enable cloudresourcemanager.googleapis.com` for project-level IAM
`gcloud services enable ml.googleapis.com` for ML Engine
`gcloud services enable spanner.googleapis.com` for Spanner
`gcloud services enable sqladmin.googleapis.com` for SQL

**Comment by maqiuyujoyce** [comment]
So will we disable the APIs for the users when they do uninstall?

**Comment by mihnjong** [comment]
No. We should not disable any API. Disabling APIs could break customers' running services.

**Comment by maqiuyujoyce** [comment]
Yeah, but I worried that users may not expect so many enabled APIs after uninstallation.

**Comment by martinmaly** [comment]
If enabling of the API(s) fails, we need to print a good error message so that users know which APIs were not successfully enabled.

**Comment by martinmaly** [comment]
Fixed

**Comment by mihnjong** [comment]
More than 5 mins, `sc add-gcp-broker` hangs only with the following message:
```
sc add-gcp-broker
using project:  [[PROJECT ID]]
```

Since it takes long to enable APIs, it would be better to print the status whenever trying to enable a API.

**Comment by maqiuyujoyce** [comment]
Maybe make the enabling process run in parallel will help?

**Comment by mihnjong** [comment]
Let me create a new issue about parallel API enablement.
